src/Components/Scenes/Governance/GovernorsTable.tsx

```diff
@@ -4,23 +4,33 @@ import TableWrapper from '../../Presentational/Table/TableWrapper';
 import Modal from '../../Presentational/Modal/Modal';
 import {
   governorsRequestFetch,
-  deleteGovernorsRequestFetch
+  deleteGovernorsRequestFetch,
+  addGovernorsRequestFetch
 } from '../../../State/Governance/actions';
+import { TextField } from '@material-ui/core/';
 
-const GovernorsTable = (props: any) => {
+const GovernorsTable = ({
+  isModalOpenAddGovernor,
+  setIsModalOpenAddGovernors
+}: any) => {
   const { governors, setSelectedRemoveRowId } = useSelector(
     (state: any) => state.governance
   );
   const titleText = `Governors`;
   const columns = [{ name: 'Governor', property: 'username' }];
   const [isModalOpen, setIsModalOpen] = useState(false);
+  const [userName, setUserName] = useState('');
```

Comment:
could we have username, createdByName, userEmail in the same object? If not, no big deal

Reply:
and also, why not place this in redux and not local state just in case?

Reply:
I will be adding into redux on my next PR